package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Organization tool for casual gatherings.",
   "main": "src/app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/app.js"
```

Comment:
I seem to recall playing with this line to try to get Heroku working. Did you add this or is it a remnant of an older pull? Either way, it shouldn't be necessary. Nodemon should use "main" I think, and Heroku uses the Procfile.